src/main/java/mat/server/CQLConstantServiceImpl.java

```diff
@@ -10,31 +10,34 @@
 import java.util.List;
 import java.util.Map;
 
+import mat.client.shared.*;
 import org.cqframework.cql.cql2elm.SystemModelInfoProvider;
 import org.hl7.elm_modelinfo.r1.ClassInfo;
 import org.hl7.elm_modelinfo.r1.ClassInfoElement;
 import org.hl7.elm_modelinfo.r1.ModelInfo;
 import org.hl7.elm_modelinfo.r1.TypeInfo;
+import org.json.JSONArray;
 import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
 import org.springframework.stereotype.Service;
 
 import com.google.gson.Gson;
 
 import mat.DTO.DataTypeDTO;
 import mat.DTO.UnitDTO;
 import mat.client.cqlconstant.service.CQLConstantService;
-import mat.client.shared.CQLConstantContainer;
-import mat.client.shared.CQLTypeContainer;
-import mat.client.shared.MatContext;
-import mat.client.shared.QDMContainer;
 import mat.dao.clause.QDSAttributesDAO;
 import mat.model.cql.CQLKeywords;
 import mat.server.service.CodeListService;
 import mat.server.service.MeasureLibraryService;
 import mat.server.util.MATPropertiesService;
 import mat.server.util.QDMUtil;
 import mat.shared.cql.model.FunctionSignature;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.web.client.RestClientResponseException;
+import org.springframework.web.client.RestTemplate;
 
+@ContextConfiguration(classes = Application.class)
```

Comment:
are you sure you need ContextConfiguration? first of all it resides in a test package and should not be available in tomcat

Reply:
I am using it in retrieving external API hostname and port from MAT.properties.

Reply:
it's a pure test only annotation. should not be part of PROD classpass

Reply:
Created a separate service class